spotlight-client/src/contentModels/transforms.ts

```diff
@@ -0,0 +1,338 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2020 Recidiviz, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+
+/**
```

Comment:
This file is large enough and covering enough ground that I wonder if it would be beneficial to structure it differently. One way would be to decompose it into `src/contentModels/transforms/*.ts` with different files dedicated to different kinds of transformations, e.g. separated by compartment (supervision vs prison). Or perhaps it would be helpful to just ensure there are docstrings on each function?

Reply:
If I were going to break this up, I think what I would want to do is make one module per record type, which would export the type definition and all of the transform functions that output that type? I'll give that a shot

Reply:
See latest update, I reorganized these functions along with the metric fetch function into a "metricsApi" directory that owns all the code related to dealing with API responses. Feels like a good change to me

Reply:
Yeah I like how this turned out!